apps/mobile/src/screens/HomeScreen/CuratedScreen.tsx

```diff
@@ -0,0 +1,160 @@
+import { Suspense, useCallback, useMemo, useState } from 'react';
+import { graphql, useFragment, useLazyLoadQuery, usePaginationFragment } from 'react-relay';
+
+import { CuratedScreenFragment$key } from '~/generated/CuratedScreenFragment.graphql';
+import { CuratedScreenGlobalFragment$key } from '~/generated/CuratedScreenGlobalFragment.graphql';
+import { CuratedScreenQuery } from '~/generated/CuratedScreenQuery.graphql';
+import { CuratedScreenTrendingFragment$key } from '~/generated/CuratedScreenTrendingFragment.graphql';
+import { RefetchableCuratedScreenGlobalFragmentQuery } from '~/generated/RefetchableCuratedScreenGlobalFragmentQuery.graphql';
+import { RefetchableCuratedScreenTrendingFragmentQuery } from '~/generated/RefetchableCuratedScreenTrendingFragmentQuery.graphql';
+import { removeNullValues } from '~/shared/relay/removeNullValues';
+
+import { FeedList } from '../../components/Feed/FeedList';
+import { LoadingFeedList } from '../../components/Feed/LoadingFeedList';
+
+type CuratedScreenInnerProps = {
+  queryRef: CuratedScreenFragment$key;
+};
+
+const PER_PAGE = 20;
+
+function CuratedScreenInner({ queryRef }: CuratedScreenInnerProps) {
+  const query = useFragment(
+    graphql`
+      fragment CuratedScreenFragment on Query {
+        ...CuratedScreenTrendingFragment
+        ...CuratedScreenGlobalFragment
+
+        ...FeedListQueryFragment
+      }
+    `,
+    queryRef
+  );
+
+  const globalFeed = usePaginationFragment<
+    RefetchableCuratedScreenGlobalFragmentQuery,
+    CuratedScreenGlobalFragment$key
+  >(
+    graphql`
+      fragment CuratedScreenGlobalFragment on Query
+      @refetchable(queryName: "RefetchableCuratedScreenGlobalFragmentQuery") {
+        globalFeed(before: $curatedFeedBefore, last: $curatedFeedCount)
+          @connection(key: "CuratedScreenFragment_globalFeed") {
+          edges {
+            node {
+              __typename
+
+              ...FeedListFragment
+            }
+          }
+        }
+      }
+    `,
+    query
+  );
+
+  const curatedFeed = usePaginationFragment<
+    RefetchableCuratedScreenTrendingFragmentQuery,
+    CuratedScreenTrendingFragment$key
+  >(
+    graphql`
+      fragment CuratedScreenTrendingFragment on Query
+      @refetchable(queryName: "RefetchableCuratedScreenTrendingFragmentQuery") {
+        curatedFeed(before: $curatedFeedBefore, last: $curatedFeedCount)
```

Comment:
oh can we add `includePosts` if the feature is enabled here? follow up PR is good!

Reply:
Yup! Added it